alpha/power.cpp

```diff
@@ -55,6 +56,7 @@ void _TS_POWER::lp_on_trans_hp()
 void _TS_POWER::hp_on_enter() {
   log_i("- Enter HP -");
   current_state = TS_PowerState::HIGH_POWER;
+  TS_RADIO.wifi_connect();
```

Comment:
how about when it leaves hp mode?

Reply:
maybe lets not put the wifi connect here but in the task with bt. if hp mode and not connected, try to connect every 1 minute (if not connected). That task would be in charge of anything radio related.

Reply:
> @fustilio are you going to address the changes or should I try? I think I'll need some pointers on how to put the retry loop in the task?
> Regarding this comment: "our retry loop is in the task, so lets not retry here"
> and to convert the task sleep to be asynch-compliant
> Regarding this comment: "async code, don't sleep here"

I'll just quickly fix them up :p